src/app/App.js

```diff
@@ -116,6 +126,44 @@ export default function App() {
             </>
           )}
         />
+        <Route
+          exact
+          path="/feedback"
+          render={() => (
+            <>
+              <Header title={'Feedback'} />
```

Comment:
you can think about implementing the header in the current page. it makes the route a little clearer because you only have one component. depends on what you prefer

Reply:
I asked a question in Slack, I think it is clearer to have it in the `App.js`